utils/generateSvgComponents.js

```diff
@@ -0,0 +1,31 @@
+/*eslint-disable */
+const fs = require('fs');
```

Comment:
this is a hack, reads the svg files from the quran.com repo and turns all surahTitle into react svg components. Not proud of this, we only have to do it once.